TFlippy_TerritoryControl_Structures_Dev/Entities/Structures/Turret/Sentry.as

```diff
@@ -276,6 +342,50 @@ void onCommand(CBlob@ this, u8 cmd, CBitStream @params)
 		this.getSprite().PlaySound(state ? "Security_TurnOn" : "Security_TurnOff", 0.30f, 1.00f);
 		this.set_bool("security_state", state);
 	}
+	else if (cmd == this.getCommandID("addAmmo"))
+	{
+
+		//mat_gatlingammo
+		u16 blobNum = 0;
+		if (!params.saferead_u16(blobNum))
+		{
+			warn("addAmmo");
+			return;
+		}
+		CBlob@ blob = getBlobByNetworkID(blobNum);
+		if (blob is null) return;
+
+		CInventory@ invo = blob.getInventory();
+		if (invo !is null)
+		{
+			int ammoCount = invo.getCount("mat_gatlingammo");
+
+			if (ammoCount > 0)
+			{
+				this.Sync("ammoCount", true);
+				this.add_u16("ammoCount", ammoCount);
```

Comment:
should this not be set_u16 since your setting the new ammo count? or is the variable name incorrect